### PR TITLE
Generate resolvers for Pathom3 from :com.wsscode.pathom3.connect.operation/resolve keys

### PR DESCRIPTION
**Pull request by danieroux**
Tony,

Here is a (currently) unsquashed PR to add `rad.resolvers-pathom3/generate-resolvers`.

- I introduced `rad.resolvers-common/secure-resolver` to keep that common
- This PR does not use `requiring-resolve` - if you include this new ns the assumption is you have Pathom3

### CODE REVIEW

File: src/main/com/fulcrologic/rad/resolvers_pathom3.cljc [hunk 1]
```diff
@@ -0,0 +1,45 @@
+(ns com.fulcrologic.rad.resolvers-pathom3
+  (:require
+    [com.fulcrologic.rad.attributes :as attr]
+    [com.fulcrologic.rad.resolvers-common :as resolvers-common]
+    ; Pathom3 is not a dependency of fulcro-rad, and should not be
```
Comment (awkay) [inline src/main/com/fulcrologic/rad/resolvers_pathom3.cljc:5]: Not a bad idea to put a ns docstring on this ns, and perhaps explain general usage along with this comment.

File: src/main/com/fulcrologic/rad/resolvers_pathom3.cljc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        (keys m)))
+
+(defn attribute-resolver
+  "Generate a resolver for an attribute that specifies a :com.wsscode.pathom3.connect.operation/resolve key. Returns a resolver
+  or nil.
+
+  Accepts any keys in the com.wsscode.pathom3.connect.operation ns,
+  thus any keys defresolver accepts are valid."
+  [attr]
+  (enc/when-let [resolver        (::pco/resolve attr)
```
Comment (awkay) [inline src/main/com/fulcrologic/rad/resolvers_pathom3.cljc:26]: What do you think about adding this key to the attributes-options ns? Like I did for the pathom 2 keys. Lets ppl know what is available, along with a bit about how to use it when they are defining attributes and using autocomplete.

**Review by awkay — CHANGES_REQUESTED** [review]

File: src/main/com/fulcrologic/rad/resolvers_pathom3.cljc (hunk 1 above)
Comment (danieroux) [reply src/main/com/fulcrologic/rad/resolvers_pathom3.cljc]: I have written a ns docstring, thank you for the suggestion.

File: src/main/com/fulcrologic/rad/resolvers_pathom3.cljc (hunk 2 above)
Comment (danieroux) [reply src/main/com/fulcrologic/rad/resolvers_pathom3.cljc:50]: I have added `pathom3-resolve`, `pathom3-output`, `pathom3-input`, and `pathom3-transform`.

Mostly copying & pasting from what was there, and adding notes on `::auth/redact` post-processing the results and `pathom3-output` generated by default.

**Comment by awkay** [comment]
Nicely done! Thanks a bunch!

**Comment by awkay** [comment]
Released in 1.2.0

**Pull request merged**